Services/CO.CDP.EntityVerification/appsettings.json

```diff
@@ -10,13 +10,18 @@
     "ServiceURL": "http://localhost:4566",
     "AuthenticationRegion": "",
     "AccessKey": "",
-    "SecretKey":  ""
+    "SecretKey": ""
   },
   "InboundQueue": {
     "Name": "",
     "MaxNumberOfMessages": 10,
     "WaitTimeSeconds": 20
   },
+  "OutboundQueue": {
+    "Name": "",
+    "MaxNumberOfMessages": 10,
+    "WaitTimeSeconds": 20
```

Comment:
Do we use these parameters with the Outbound Queue?

Reply:
Good point, we don't. I have removed them.